src/Umbraco.Web.UI.Client/src/views/components/blockcard/umb-block-card-grid.less

```diff
@@ -1,15 +1,20 @@
 .umb-block-card-grid {
-    /* FlexBox Fallback */
-    display: flex;
-    flex-wrap: wrap;
-    > * {
-        flex: 1 1 240px;
-    }
-
-    /* Grid Setup */
     display: grid;
     grid-template-columns: repeat(auto-fill, minmax(200px, 1fr));
     grid-auto-rows: minmax(160px, auto);
     grid-gap: 20px;
+
+    // Avoid grid layout to re-arrange when opening overlay and scrollbar is hidden.
+    max-width: 99.9%;
+
+    @supports not (display: grid) {
```

Comment:
Flex fallback isn't needed since grid has been support in major browsers for five or so years. Not going to do anything about it now, but noted for later 😄

Reply:
I have removed the legacy flexbox fallback now. Furthermore `grid-gap` was a non-standard property used in older browsers. https://caniuse.com/?search=grid-gap